ambry-network/src/main/java/com.github.ambry.network/NetworkClient.java

```diff
@@ -163,7 +184,9 @@ private void handleSelectorEvents(List<ResponseInfo> responseInfoList) {
       connectionTracker.checkInConnection(connId);
     }
 
-    for (String connId : selector.disconnected()) {
+    List<String> disconnectedList = selector.disconnected();
+    networkMetrics.connectionsDisconnectedCount.mark(disconnectedList.size());
```

Comment:
this seems like it should be part of the selector metric - it there one already for this?